docs/templates/main.md

```diff
@@ -0,0 +1,128 @@
+# Templates system
+
+By default, Hybrid Container Cloud delivers the set of default `Template` objects. It's also supported to build your
+own templates and use it for the deployment.
```

Comment:
Let's mention, that templates can only reside in the hmc-system namespace at the moment, but can be referenced by Deployments from any namespace.